tools/crm_mon.c

```diff
@@ -297,6 +297,7 @@ struct {
     { "summary", pcmk_section_summary },
     { "tickets", pcmk_section_tickets },
     { "times", pcmk_section_times },
+    { "verifications", pcmk_section_verify },
```

Comment:
This also needs to be added to two error messages -- search crm_mon.c for " options:"